src/dialog-holder.component.ts

```diff
@@ -79,12 +82,27 @@ export class DialogHolderComponent {
     }, 300);
   }
 
+  /**
+   * Bind a body class 'modal-open' to a condition of dialogs in pool > 0
+   */
+  
+  private updateBodyClass() {
+    const bodyClass = 'modal-open';
+    const body = document.getElementsByTagName('body')[0];
+    if(this.dialogs.length == 0) {
```

Comment:
I would use `===` to check this. Personal preference. Or even `if(!this.dialogs.length)`